### ISSUE TITLE
xCode 7 and last version of Swift

### ISSUE THREAD

**Issue by ghost** [yomajkel/RingGraph#7]
Hello,

I'm just trying to use your code but it's looks not compatible with the last versions of xCode and Swift. Any idea ?

Regards

**Comment by 36bydesign** [comment]
Agreed, it's not even close to compatible. This is an abandoned project.

I'm looking into using GuageKit, looks promising and I was able to launch the demo app on my phone, checks out.
https://github.com/skywinder/GaugeKit
Also...
https://github.com/maxkonovalov/MKRingProgressView

**Comment by kremich** [comment]
Sorry to miss this issue. I didn't know people are using this library. I'll try to jump in today and make it work.

**Comment by kremich** [comment]
I pushed the changes to master and it should work if you pull the repo directly. I'll try to update pod spec tomorrow so it will also work as a cocoa pod again.

**Comment by 36bydesign** [comment]
Works like a charm now, thanks so much yomajkel, great work.

**Comment by ghost** [comment]
I try to use cocoapod but it always use older version 0.2. It's not working in xCode 7 :-(

```
[!] Unable to satisfy the following requirements:

- `RingGraph (~> 0.3.0)` required by `Podfile`
```

If I don't set a version in PodFile, the version dowloaded is 0.2.4. You may update something in Cocoapod (please)

**Comment by ghost** [comment]
I find a temporary solution. In Podfile use this line :

 `pod 'RingGraph', :git => 'https://github.com/yomajkel/RingGraph.git', :tag => '0.3.0'
`

Maybe you may do something for cocoapod community.

**Comment by kremich** [comment]
Hey guys, I forgot to push specs for 0.3.0. I'm sorry for that. The workaround from @iLandes will work as well. Now you should be able to just get a 0.3.0 pod.

**Comment by ghost** [comment]
The pod install with just : `pod 'RingGraph', '~> 0.3'` still not working this morning 08/01/16

**Comment by kremich** [comment]
@iLandes I just tried and it works for me. Also I see version 0.3.0 on cocoapods.org website. Maybe you can retry as some caches needed to refresh.
